addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -688,14 +693,26 @@ export class PosStore extends Reactive {
         // This actions cannot be handled inside pos_order.js or pos_order_line.js
         if (values.product_id.to_weight && this.config.iface_electronic_scale && configure) {
             if (values.product_id.isScaleAvailable) {
-                const weight = await makeAwaitable(this.env.services.dialog, ScaleScreen, {
-                    product: values.product_id,
-                    taxIncluded: this.config.iface_tax_included === "total",
-                });
+                this.isScaleScreenVisible = true;
```

Comment:
can you use `useTrackedAsync` in order to automatically keep track of this state?